core/tee/tee_svc.c

```diff
@@ -31,23 +31,24 @@ vaddr_t tee_svc_uref_base;
 
 void syscall_log(const void *buf __maybe_unused, size_t len __maybe_unused)
 {
-#ifdef CFG_TEE_CORE_TA_TRACE
-	char *kbuf;
+	if (IS_ENABLED(CFG_TEE_CORE_TA_TRACE)) {
```

Comment:
can remove the `__maybe_unused` attributes at line 32.

Reply:
if CFG_TEE_CORE_TA_TRACE not define, len maybe unused.

Reply:
@xiaoxuZeng just to clarify, `__maybe_unused` is needed when there is conditional compilation (`#ifdef CFG_SOMETHING`) but not when `IS_ENABLED(CFG_SOMETHING)` is used, because the later if effectively an `if ()` construct so the compiler will see the variables used inside the block even if the condition is false (which is why we prefer using `IS_ENABLED()` whenever possible).

Reply:
Got， Thanks very much!

Reply:
> @xiaoxuZeng just to clarify, `__maybe_unused` is needed when there is conditional compilation (`#ifdef CFG_SOMETHING`) but not when `IS_ENABLED(CFG_SOMETHING)` is used, because the latter if effectively an `if ()` construct so the compiler will see the variables used inside the block even if the condition is false (which is why we prefer using `IS_ENABLED()` whenever possible).

Got， Thanks very much!